libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/reststop/RestStop.kt

```diff
@@ -10,13 +10,15 @@ import com.mapbox.navigation.base.trip.model.roadobject.location.RoadObjectLocat
  *
  * @param restStopType information about a rest stop. See [RestStopType].
  * @param name name of the rest stop.
+ * @param amenities facilities associated with the rest stop. See [AmenityType]
  * @see RoadObject
  * @see RoadObjectType.REST_STOP
  */
 class RestStop internal constructor(
     id: String,
     @RestStopType.Type val restStopType: Int,
     val name: String?,
+    val amenities: List<Amenity>?,
```

Comment:
add field `amenities` to `equals`, `hasCode`, `toString`

Reply:
```suggestion
    val amenities: List<Amenity>,
```

Reply:
This will be nullable as `amenities` is optional in directions response. @RingerJK